src/Symfony/Component/Messenger/Handler/MessageHandlerInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Handler;
+
+/**
+ * Handlers can implement this interface. This allows them to be auto-configured and to
```

Comment:
I would remove the mention of autoconfiguration.
The interface is useful on its own to add semantic to an invokable class.
THEN, yes, autoconfiguration can leverage this added semantic. But that's independent.

Reply:
Changed 👍